proto/internal/uber/cadence/history/v1/service.proto

```diff
@@ -274,6 +274,8 @@ message ResetWorkflowExecutionResponse {
 message TerminateWorkflowExecutionRequest {
   api.v1.TerminateWorkflowExecutionRequest request = 1;
   string domain_id = 2;
+  api.v1.WorkflowExecution external_workflow_execution = 3;
+  bool child_workflow_only = 4;
```

Comment:
Can you add a comment for this field? (also would be nice for other `child_workflow_only` similarly). 
This name makes me think it's terminating the childWFs of the wf in the request. I think having comments will help a lot.

Reply:
Done. Added comments for signal, cancel and terminate.